pyomo/solvers/plugins/solvers/cplex_direct.py

```diff
@@ -66,6 +66,7 @@ def _init(self):
             while len(self._version) < 4:
                 self._version += (0,)
             self._version = self._version[:4]
+            self._version = tuple([int(i) for i in self._version])
```

Comment:
Can you combine this with the line before:
```
self._version = tuple(int(i) for i in self._version[:4])
```

Reply:
Done.